docusaurus/src/remark/docsHeaderDecoration.js

```diff
@@ -63,27 +62,35 @@ const buildConnectorHTMLContent = (
   // note - you can't have any leading whitespace here
   const htmlContent = `<div>
     <div class="header">
-      <img src="${
-        registryEntry.iconUrl_oss
-      }" alt="connector logo" style="max-height: 40px; max-width: 40px; float: left; margin-right: 10px" />
-      <h1 id="${originalId}" style="position: relative;">${originalTitle}</h1>
+      <img src="${registryEntry.iconUrl_oss}" alt="" class="connectorIcon"  />
```

Comment:
:information_source: Images should only have an alt text if it provides useful information about the image. If the image is purely decorative (as in this case), prefer an empty `alt` tag, which causes the browser to treat it as invisible to assistive technologies, which is the desired behavior.